### PR TITLE
updatePermissions API fix - 2027

### PR DESCRIPTION
**Pull request by priyanka-TL**

### CODE REVIEW

**Review by adithyadinesh0412 — APPROVED** [review]
Reviewed on Nov 25 - 1.24PM

File: src/services/permissions.js [hunk 1]
```diff
@@ -92,7 +92,15 @@ module.exports = class PermissionsHelper {
 				})
 			}
 		} catch (error) {
-			throw error
+			if (error.name === common.SEQUELIZE_DATABASE_ERROR && error.parent?.code === '22003') {
```
Comment (aks30) [inline src/services/permissions.js]: @priyanka-TL can we not check whether its integer or not in validator only?

File: src/services/permissions.js [hunk 2]
```diff
@@ -125,7 +133,15 @@ module.exports = class PermissionsHelper {
 				result: {},
 			})
 		} catch (error) {
-			throw error
+			if (error.name === common.SEQUELIZE_DATABASE_ERROR && error.parent?.code === '22003') {
```
Comment (aks30) [inline src/services/permissions.js]: @priyanka-TL can we not check whether its integer or not in validator only?

**Review by aks30 — CHANGES_REQUESTED** [review]
Reviewed on 25 Nov

File: src/services/permissions.js (hunk 1 above)
Comment (priyanka-TL) [reply src/services/permissions.js]: done

**Pull request merged**